src/DynamoCore/DynamoCore.csproj

```diff
@@ -137,6 +128,7 @@
       <NodeHelpFiles Include="$(SolutionDir)..\doc\distrib\NodeHelpFiles\**\*.*" />
       <OpenSourceLicenses Include="$(SolutionDir)..\doc\distrib\Open Source Licenses\**\*.*" />
       <LocalizedResources Include="$(SolutionDir)..\extern\Localized\**\*.*" />
+      <LegacyBinariesToBinFolder Include="$(SolutionDir)..\extern\legacy_remove_me\bin\*\*" />
```

Comment:
what is this change for?

Reply:
I thought @aparajit-pratap just removed something similar?

Reply:
I don't know how that got there. Will revert this